kyverno/helm/kyverno/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
+name: kyverno
+description: A Helm chart for Kubernetes
+type: application
+version: 0.1.0
```

Comment:
@tikr7 Also bump this chart version to `0.1.1` please.

Reply:
done